content/en/docs/clients/restful-api/rebuild.md

```
@@ -11,7 +11,7 @@ weight: 6
 ##### Method & Url
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)